src/simparams.jl

```diff
@@ -42,10 +44,14 @@ struct SimParams
   phone_tracing_params::Union{Nothing, PhoneTracingParams}
 
   infection_modulation_function::Union{Nothing,InfectionModulation}
+  infection_travels_function::Union{Nothing,InfectionTravels}
+  travels_frequency::TimePoint
```

Comment:
this probably should be a part of the travel function (different functions should be allowed do have different parameters)

Reply:
function progression is independent of this parameter. function outsidefeed!  push the events in the queue depending on the frequency